inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/InLongMessageHandler.java

```diff
@@ -44,30 +62,6 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.nio.charset.StandardCharsets;
```

Comment:
It seems that your code style in your local IDE was changed.

Please refer to this: https://github.com/apache/inlong/discussions/3082 and reformat the import order, thanks.